project/AkkaDisciplinePlugin.scala

```diff
@@ -34,15 +41,28 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
       import sbt.librarymanagement.{ SemanticSelector, VersionNumber }
       !VersionNumber(scalaVersion.value).matchesSemVer(SemanticSelector("<=2.11.1"))
     })
+
+  val silencerVersion = "1.3.1"
+  lazy val silencerSettings = Seq(
+    libraryDependencies ++= Seq(
+      compilerPlugin("com.github.ghik" %% "silencer-plugin" % silencerVersion),
+      "com.github.ghik" %% "silencer-lib" % silencerVersion % Provided,
+    )
+  )
   
   lazy val disciplineSettings =
     scalaFixSettings ++
+    silencerSettings ++
     scoverageSettings ++ Seq(
       Compile / scalacOptions ++= (if (strictProjects.contains(name.value)) {
                                  disciplineScalacOptions
                                } else {
                                  disciplineScalacOptions -- undisciplineScalacOptions
                                }).toSeq,
+      Compile / scalacOptions ++= (
+        if (fatalWarningsFor(name.value)) Seq("-Xfatal-warnings")
```

Comment:
so distributed data can be fatal but doesn't work with all the other ones yet?

Reply:
Correct!